app/models/involved_person.rb

```diff
@@ -37,10 +37,18 @@ class InvolvedPerson
       validates :injury_level, inclusion: { in: INJURY_LEVELS }
       validates :injury_type, presence: true, subset: { in: INJURY_TYPES }
       validates :medical_aid, inclusion: { in: MEDICAL_AID }
-      validates :injury_from_preexisting_condition, inclusion: { in: [true, false], message: Constants::ERROR_BLANK_FIELD }
+
+      with_options unless: :medical_aid_refused? do
+        validates :injury_from_preexisting_condition, inclusion: { in: [true, false],
+                                                                   message: Constants::ERROR_BLANK_FIELD }
+      end
     end
   end
 
+  def medical_aid_refused?
+    medical_aid.eql?("No medical assistance or refused assistance")
```

Comment:
In general, we use `==` instead of `eql?` for things like string comparison, but that's just our convention and doesn't make a difference one way or another.